docs/source/markdown/podman-image-sign.1.md

```diff
@@ -9,7 +9,8 @@ podman-image-sign - Create a signature for an image
 ## DESCRIPTION
 **podman image sign** will create a local signature for one or more local images that have
 been pulled from a registry. The signature will be written to a directory
-derived from the registry configuration files in /etc/containers/registries.d. By default, the signature will be written into /var/lib/containers/sigstore directory.
+derived from the registry configuration files in /etc/containers/registries.d for root user,
+or $HOME/.config/containers/registries.d for non-root user. By default, the signature of root user will be written into /var/lib/containers/sigstore directory.
```

Comment:
Where will signatures be written for non root users?

$HOME/.local/share/containers/sigstore?

Reply:
non-root users should create their YAML files under the $HOME/.config/containers/registries.d. Do we provide the default config file like default.yaml?

Reply:
Non root users are not going to know how to do this, so we need to setup good defaults.  podman should automatically detect that there is no default.yaml, and create it and store signatures in $HOME/.local/share/containers/sigstore.  If default.yaml exists, then it will follow those rules.  I would guess.

Reply:
Set the default sigstore to `$HOME/.local/share/containers/sigstore` if the config file is using `/var/lib/containers/sigsore`. Also hard code this path if rootless user has no configurations at all.